### PR TITLE
Add NuGet download instructions to docs

### PR DESCRIPTION
**Pull request by supergibbs**
I will maintain the NuGet packages and ensure they stay up to date until we can automate the process

### CODE REVIEW

File: docs/getting-started/download.md [hunk 1]
```diff
@@ -69,6 +69,17 @@ You can also install and manage Bootstrap's Sass and JavaScript using [Composer]
 $ composer require twbs/bootstrap
 {% endhighlight %}
 
+### NuGet
+
+If you develop in .NET you can also install and manage Bootstrap's [CSS](https://www.nuget.org/packages/bootstrap/) or 
+[Sass](https://www.nuget.org/packages/bootstrap.sass/) and JavaScript using [NuGet](https://www.nuget.org):
+
+{% highlight bash %}
```
Comment (XhmikosR) [inline docs/getting-started/download.md]: Should't this be `highlight powershell`?

**Comment by XhmikosR** [comment]
So, @supergibbs can you make the change I mention [here](https://github.com/twbs/bootstrap/pull/17214#discussion_r38811360)?

Rouge 1.9-something added Powershell highlighting.

**Comment by supergibbs** [comment]
Good call @XhmikosR, I've updated it.

**Comment by XhmikosR** [comment]
@supergibbs: can you squash those patches?

**Comment by supergibbs** [comment]
OK squashed and rebased on `v4-dev`

**Comment by cvrebert** [comment]
https://github.com/twbs/bootstrap/blob/v4-dev/README.md#quick-start also needs a corresponding update

**Comment by supergibbs** [comment]
@cvrebert done

**Comment by cvrebert** [comment]
Made a couple tweaks and merged this as f5ab82c7eb4a26bac305333df9a33860ac91e1d7.
Thanks!

**Pull request closed without merging**